site/content/extend/plugins/best-practices.md

```diff
@@ -6,6 +6,67 @@ weight: 50
 
 See here for [server-specific best practices for plugins](/extend/plugins/server/best-practices/). Webapp-specific best practices are incoming.
 
+## How can I create my own plugin setting?
+
+Plugins can define their admin settings in the plugin's manifest [setting schema](https://developers.mattermost.com/extend/plugins/manifest-reference/#settings_schema) which include a pre-defined types that the System Console UI currently supports. 
+
+Additionally, a plugin could define their own type of setting with a corresponding custom user interface that will be displayed in the system console following the process below. 
```

Comment:
As per my previous comment, and here's my suggestion: 
```suggestion
Alternatively* a plugin can contain a custom setting, and a corresponding custom user interface will be displayed in the System Console. 
```
* Assuming the plugin can only be one or the other? Custom or pre-defined.

Reply:
A plugin can have both pre-defined and custom settings. I still believe your correction is valid: 

> Alternatively a plugin can contain a custom setting, and a corresponding custom user interface will be displayed in the System Console.

@justinegeffen Do you agree or would you change something more there?

Reply:
I agree with that, I think it still works. :)